### ISSUE TITLE
Loading AIST++ data

### ISSUE THREAD

**Issue by Jhc-china** [lisiyao21/Bailando#21]
When training the CrossCondGPT2 using `cc_motion_gpt.yaml` config and loading the AIST++ data using `load_data_aist`[line 538] in `./utils/functional.py`, I'm confusing to the for loop in line 572.
Because of 8 downsample rate, the **np_music** feature current read is at 7.5fps, while the **np_dance** is at 60fps. However, in the for loop, the `seq_len` is associate with **np_music**, and the step for `range(...)` is _move_(which is 8 in the config yaml), the step for **np_music** is i/music_sample_rate (the music_sample_rate I think is also 8), the step for **np_dance** is 8. As a result, the music training sample is actually step by 1 frame in the `range(...)` loop. If the music sample step by 1 frame, we should get `len(np_music)` training sample. But the step in `range(....)` is 8, `len(np_music) - len(np_music)//8` training sample is left behind.
In a nutshell, I think the split function in for loop doesn't match.
